src/components/Footer/Footer.jsx

```diff
@@ -1,52 +1,60 @@
-import React,{useEffect, useState} from "react";
+import React, { useEffect, useState } from "react";
 import "./Footer.css";
 
-const Footer = ({props}) => {
+const Footer = ({ currentPage, rowsPerPage, setRowsPerPage, SetCurrentPage, customersToRender, customersReadyToRender }) => {
+    const countActiveCustomers = (customersList) => {
+        return customersList.filter(customer => customer.status === "active").length;
+    }
 
-let  [currentPage,rowsPerPage,setRowsPerPage,SetCurrentPage,customersToRender,customersReadyToRender] = props;
+    const [activeCustomers, setActiveCustomers] = useState(countActiveCustomers(customersToRender));
 
-const countActiveCustomers = (customersList) => {
-    return customersList.filter(customer=>customer.status==="active").length;
-}  
+    const handleChange = (e) => {
+        const rows = parseInt(e.target.value);
+        SetCurrentPage(1);
+        setRowsPerPage(rows);
+    };
 
-const [activeCustomers,setActiveCustomers] = useState(countActiveCustomers(customersToRender));
+    const handlePreviousPage = () => {
+        SetCurrentPage(previous => previous > 1 ? previous - 1 : previous);
+    };
 
-const handleChange = (e) => {
-    SetCurrentPage(1);
-    setRowsPerPage(parseInt(e.target.value));
-};
+    const handleNextPage = () => {
+        SetCurrentPage(previous => customersToRender.length > rowsPerPage * currentPage ? previous + 1 : previous);
+    };
 
-const handlePreviousPage = () => {
-    SetCurrentPage(previous => previous > 1 ? previous-1 : previous);
-};
-
-const handleNextPage = () => {
-    SetCurrentPage(previous => customersToRender.length > rowsPerPage * currentPage ? previous + 1 : previous);
-};
-
-useEffect(() => {
-    setActiveCustomers(countActiveCustomers(customersToRender));
-},[customersToRender])
+    useEffect(() => {
+        setActiveCustomers(countActiveCustomers(customersToRender));
+    }, [customersToRender])
 
     return (
         <footer>
             <div className="active-customers">
-                active customers: <strong>{activeCustomers}</strong> / <small>{customersToRender.length}</small>
+                active customers:
+                {` `}
+                <strong>{activeCustomers}</strong>
+                {` `} / {` `}
+                <small>{customersToRender.length}</small>
             </div>
             <div className="right-side">
                 <ul>
                     <li className="rows-page">
                         Rows per page:
                         <select onChange={handleChange}>
                             <option value="5">5</option>
-                            <option value="10">10</option>
+                            <option value="10" selected>10</option>
                             <option value="15">15</option>
                             <option value="20">20</option>
                             <option value="25">25</option>
                         </select>
```

Comment:
I would recommend you to create a component for select.
We might need it somewhere else